src/test/java/org/openmrs/contrib/qaframework/page/AttachmentsPage.java

```diff
@@ -35,6 +42,17 @@ public void setFileUrl(String fileUrl) {
 		element.sendKeys(fileUrl);
 	}
 
+	public String createTemporaryFile() throws IOException {
```

Comment:
@Irenyak1 what do you think of `createTemporarlyFile()` vs `createRandomFile()`?

Reply:
@kdaud since it is a temporary file that will be created, I felt it would be better to name it accordingly.  What do you think?

Reply:
@Irenyak1 I see your point is fair enough. However, is this file destroyed any where after the test run?

Reply:
> 

No @kdaud, the file is not destroyed.

Reply:
> the file is not destroyed.

then why call the method `creatTemporarlFile()`?